### PR TITLE
Add a common entry point for all benchmarks

### PR DESCRIPTION
**Pull request by radeusgd**
### Pull Request Description

Implements https://www.pivotaltracker.com/story/show/183915527

Adds a common entry point to the `Benchmarks` test suite, so that all benchmarks can be run at once.

Usually we are running only one set of benchmarks that we want to measure, but for the purpose of checking if they are still compiling after a language change, it can be useful to run all of them (currently this takes a pretty long time though, but may work when going for lunch break :)).

Related to https://www.pivotaltracker.com/story/show/183369386/comments/234258671

### Checklist

Please include the following checklist in your PR:

- [x] The documentation has been updated if necessary.
- [ ] All code conforms to the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides.
- All code has been tested:
  - [ ] Unit tests have been written where possible.
  - [ ] If GUI codebase was changed: Enso GUI was tested when built using BOTH
        `./run ide build` and `./run ide watch`.

### CODE REVIEW

**Comment by radeusgd** [comment]
Errors like these lead me to think if we should add the (pure Enso) benchmarks to some CI step, as otherwise we may keep breaking them like here. 

OTOH statics was one of the bigger language changes, usually the changes will rarely be breaking like that. So maybe we should just be more careful to run all benchmarks upon such changes, knowing that they are not tested automatically (we should probably have a single entry point that runs all of them).

Alternatively, to not make CI too slow maybe we could add some env var which will limit each benchmark to one iteration only just to check 'compilation'.

**Review by jdunkerley — APPROVED** [review]

**Comment by radeusgd** [comment]
I've encountered a bug while trying to run the tests, as described in this Discord thread:
https://discord.com/channels/401396655599124480/1041812546732171294

It may be reasonable to wait with this PR until we agree what is the sensible resolution to that bug (so I can either wait for a fix or move to a slightly ugly workaround).

**Review by JaroslavTulach — COMMENTED** [review]
How shall one run just a single benchmark? When I am using IGV, I'd like to run as little code as possible - e.g. preferably just a single suite. What is the recommended way to do it?

**Comment by radeusgd** [comment]
> How shall one run just a single benchmark? When I am using IGV, I'd like to run as little code as possible - e.g. preferably just a single suite. What is the recommended way to do it?

This has not changed in this PR - all the time we've been running a single script to run the specific benchmark.

So for example, I'd do `enso run test/Benchmarks/src/Vector/Sort.enso` to run the sorting benchmarks. This has not changed with this PR.

The only change is the behaviour for running the project's default entry point - `enso run test/Benchmarks/`. Until now, this would run a single benchmark (sum), probably this is the first one ever developed here and it stayed under `Main.enso` but that does not make much sense. Now this will run all benchmarks (which I am aware is not the most common usecase, the usecase for that is to check that we didn't break compilation when modifying language/stdlib). To run that particular benchmark you'd need to `enso run test/Benchmarks/src/Sum.enso`.

File: test/Benchmarks/src/Json.enso [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from Standard.Base import all
+import Standard.Base
```
Comment (jdunkerley) [inline test/Benchmarks/src/Json.enso]: Feels better to amend `Base.Json` to just `Json`?

**Review by jdunkerley — COMMENTED** [review]
LGTM - one minor tweak

File: test/Benchmarks/src/Json.enso (hunk 1 above)
Comment (radeusgd) [reply test/Benchmarks/src/Json.enso]: The benchmark filename is `Json` so `Json` refers to this file. We can rename the file if needed though.

**Pull request merged**